### DISCUSSION TITLE
increment_descriptor_version missing in AbstractDescriptorProtocol

### DISCUSSION THREAD

**Discussion by pstaeck** [Draegerwerk/sdc11073#313] [General] [answered]
Is the method `increment_descriptor_version` missing in the `AbstractDescriptorProtocol`?

I run into typing issues and was wondering why this is not part of the protocol. As suggested by a colleague, type cast helps at this point but in my case this leads to unnecessarily long expression.

`clock_descriptor: descriptorcontainers.ClockDescriptorContainer = typing.cast(descriptorcontainers.ClockDescriptorContainer, mgr.get_descriptor(descriptor_handle='Clock'))`

**Comment by deichmab-draeger** [comment, marked as answer]
It is missing, but kind of intentionally. Normally there should be no need to use this method outside the internals of sdc11073. Why do you use that method?

**Comment by pstaeck** [comment]
Thank you for your answer. We are currently integrating sdc11073 version 2 into our package. In an old unit test this was used but it indeed does not seem to be necessary.
